src/components/dropdown-user.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const DropdownUser = React.memo(({id, username, onClick}) => {
```

Comment:
Better Naming ....... These are dropdown items. Also keep the prop names as generic so that such type of list can be used in multiple places